index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Athaliah's Portflio  </title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      
+          <!--Navbar-->
+          <nav id="nav">
+            <a class="logo" href="#">My Logo</a>
+            
+            <nav><img src="./Images/Union.svg" alt="burger-menu"></nav>
+               
+          </nav>
+          <section class="hero" id="main-info">
+            <h1 class="bio-intro">I’m Athaliah<br>
+             Glad to see you!</h1>
+             <p class="bio-text">
+                 I’m a software developer! I can help you build a product , 
+                 feature or website Look through some of my work and experience! 
+                 If you like what you see and have a project you need coded, don’t hestiate to contact me.
+             </p>
+             <footer>
+                 <h3>LET'S CONNECT</h3>
+                 <div id="socials">
+                    <ul>
+                      <li class="handel"><a href="https://twitter.com/AthaliahLi"><img src="./Images/twitter.svg" alt="twitter link"></a></li>
+                      <li class="handel"><a href="https://www.linkedin.com/in/athaliah-li-56217324a/"><img src="./Images/Linkedin icon.svg" alt="linked link"></a></li>
+                      <li class="handel"><a href="#"><img src="./Images/Vector.svg" alt="M"></a> </li>
+                      <li class="handel"><a href="https://github.com/Athaliah8"><img src="./Images/github.svg" alt="github link"></a></li>
+                      <li class="handel"><a href="#"><img src="./Images/angellist.svg" alt="angellist"></a></li>
+                    </ul>
+                   </div>
+             </footer>
+         </section>
+
+    <section id="work_outline">
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 The `border-radius` value you gave to the `list of languages`(e.g.: `HTML, CSS...`) in the `works` section shows them as a `hexagon`, please set it to the correct value like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Also, it seems that there is an error in the style of the `See project` button, because the design is not visible. Please correct them 🙏 

![image](https://user-images.githubusercontent.com/63915024/199776480-5fa46b98-defb-4500-a76a-8b02c148de8c.png)